src/services/validation.service.js

```diff
@@ -76,17 +77,31 @@ validator.attributes.validation = (instance, schema, options, ctx) => {
   return undefined;
 };
 
-module.exports.validate = (data) => {
+module.exports.validate = (data, isLcSubmission = false) => {
   logEmitter.emit("functionCall", "validation.service", "validate");
   const result = [];
-  const validatorResult = validator.validate(data, schema.registration);
+
+  var validationSchema = schema;
+  if (isLcSubmission) {
+    logEmitter.emit("info", "Validating with LC submission schema");
```

Comment:
Do we still need a separate schema? If so, why is the required status different for the current one - I thought needed to be all the same now?

Reply:
There is inconsistency with the validation schema for the data coming from the front-end and the OpenAPI spec.
The validation for the LA submissions is based on the OpenAPI spec, since that is the externally visible schema.